packages/sit-onyx/src/components/OnyxListbox/types.ts

```diff
@@ -15,7 +15,7 @@ export type OnyxListboxProps<TValue extends SelectionOptionValue = SelectionOpti
    */
   modelValue?: TValue;
   /**
-   * If true, the label will be visually hidden.
+   * If `true`, the label will be visually hidden and the `title` attribute will be set.
```

Comment:
the listbox does not have the title feature

Reply:
done